ngrok/src/config/http.rs

```diff
@@ -298,6 +301,19 @@ impl HttpTunnelBuilder {
         });
         self
     }
+
+    /// Configures user agent filter for this edge.
+    pub fn user_agent_filter(
+        &mut self,
+        allow: impl Into<Vec<String>>,
+        deny: impl Into<Vec<String>>,
```

Comment:
For other cases of multiples the current interface allows calling multiple times, accumulating each item. A close analog to this case is probably `allow_cidr` and `deny_cidr`, could turn this into `allow_user_agent` and `deny_user_agent`.